src/components/Kv/KvCarousel.vue

```diff
@@ -9,16 +9,16 @@
 			@touchstart="pauseOnHover ? paused = true : paused = false"
 			@mouseleave="paused = false"
 			@touchend="paused = false"
-			v-touch:swipe.left="previous"
-			v-touch:swipe.right="advance"
+			v-touch:swipe.left="previousClick"
+			v-touch:swipe.right="advanceClick"
```

Comment:
Because there are several ways to interact with this carousel - swipe left/right, click arrow left/right, click indicator button - making 1 method to that takes the index to go to and the type of interaction to we want to emit will keep this code cleaner

Reply:
To an extend I agree, but you'll need to ensure that interaction type is tagged somehow so we can maintain fidelity in the analytics.